www/common.inc

```diff
@@ -58,8 +58,25 @@ $request_context = new RequestContext($_REQUEST, $_SERVER);
 // Disable caching by default
 header("Cache-Control: no-store, no-cache, must-revalidate, max-age=0", true);
 
+// add CSP report only
+header("Content-Security-Policy-Report-Only: default-src 'self'; 
+script-src 'report-sample' 'self' https://g.3gl.net/jp/8627/v3.3.4/M https://js.braintreegateway.com/web/dropin/1.33.0/js/dropin.min.js https://ssl.google-analytics.com/ga.js;
```

Comment:
I'm wondering if we should include `unsafe-inline` for both `script-src` and `style-src` for now. If we don't this is going to get _very_ noisy right away. We can focus on fixing up the other stuff first and circle back to these after the rest of the policy is mostly set.

Maybe even a separate report-only policy at that point.

Reply:
Also, for `script-src` specifically, probably good enough to just use the origin and ditch the full path?